Data/Skills/act_dex.lua

```diff
@@ -1811,6 +1811,17 @@ skills["CorpseEruption"] = {
 	},
 	baseMods = {
 		skill("explodeCorpse", true, { type = "SkillPart", skillPart = 2 }),
+		skill("CritChance",0,{ type = "SkillPart", skillPart = 2 }),
+		skill("PhysicalMin", 0,{ type = "SkillPart", skillPart = 2 }),
+		skill("PhysicalMax",0,{ type = "SkillPart", skillPart = 2 }),
+		skill("LightningMin",0,{ type = "SkillPart", skillPart = 2 }),
+		skill("LightningMax",0,{ type = "SkillPart", skillPart = 2 }),
+		skill("ColdMin",0,{ type = "SkillPart", skillPart = 2 }),
```

Comment:
Add whitespace after commas.

Reply:
There is still one instance of incorrect formatting present in all all corpse skill related files.